.mockery.yaml

```diff
@@ -6,12 +6,18 @@ filename: "{{ .InterfaceName | snakecase }}.go"
 mockname: "{{.InterfaceName}}"
 
 packages:
-  ? github.com/mongodb/terraform-provider-mongodbatlas/internal/service/searchdeployment
```

Comment:
mockery config file was not working as expected, only the last package (in this case `advancedcluster`) was used for generating mocks.